### PR TITLE
Overhaul wiki

### PR DESCRIPTION
**Pull request by Darkhost1999**
Add more 404 CSS
Remove extra whitespace in config.yml
rename .markdown to .md
Update wording for professionalism and clarity
Upgrade the home page
Let people find "the project page" by telling them it's "the official website"
add new lines to bottoms of pages

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: Project links and build info
 
 This wiki houses documentation and install guides for the PlayStation 4 emulator RPCSX. Development happens on [our GitHub](https://github.com/rpcsx/rpcsx).
 
-You can also find more information on [the project page](https://rpcsx.github.io/rpcsx-site)
+You can also find more information on the [official RPCSX website](https://rpcsx.github.io/rpcsx-site)
```
Comment (OCDkirby) [inline about.md]: Currently the wrong link, should just be https://rpcsx.github.io

File: install-guide.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ sudo dnf install cmake libunwind-devel glfw-devel vulkan-devel vulkan-validation
 go to [Continued Install.](/wiki/installation/#continued-install)
 
   - **Arch**
-    - Install traditional dependencies. (vulkan-devel is a group, and you should install **all**!)
+    - vulkan-devel is a group, and you should install **all**!)
```
Comment (OCDkirby) [inline install-guide.md]: Remove closing parentheses

**Review by OCDkirby — CHANGES_REQUESTED** [review]

**Comment by Darkhost1999** [comment]
I gotta update 404.html

**Comment by Darkhost1999** [comment]
I'm satisfied with the changes. I couldn't think of a PR name. So I recognized I touched almost all the files. Why not say overhaul?

**Comment by OCDkirby** [comment]
I have some style changes that I'm going to add on top of the branch before merge, but other than that it's ready! Thanks :)

**Pull request merged**